tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/default-config/test-suite-a/index.html

```diff
@@ -7,7 +7,7 @@
     <section>
       <h1>test-suite-a</h1>
       <section>
-        <p><a href="./">Packages</a> > <a href="./test-suite-a/">test-suite-a</a></p>
+        <p><a href="/">Packages</a> > <a href="/test-suite-a/">test-suite-a</a></p>
```

Comment:
So Docusaurus ends up transforming this, right? Otherwise I'd expect the `Packages` link to redirect to the landing page in the public site once deployed.

Reply:
Yeah, the various site generators generally transform these site-relative-routes into fully qualified URLs. All of the ones I've worked with function that way. Still not 100% confident that this is conceptually portable, but I took this concept directly from `api-documenter`, so there's at least precedent for it.